bandit/plugins/zipfile_extractall.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding:utf-8 -*-
+#
+# Copyright (c) 2018 [disconnect3d](https://github.com/disconnect3d)
```

Comment:
Rather not have links to github in copyright statements.

Reply:
We already have such in some other file, but sure I can delete it and just leave my nickname.